docs/reference/cli.rst

```diff
@@ -173,3 +173,15 @@ Example::
 
     This command publishes drafts. You should review drafts before using this
     command, because they will become public.
+
+**********************
+Maintenance and repair
+**********************
+
+.. _fix-mptt:
+
+``fix-mptt``
```

Comment:
Perhaps a pointer to the previous version's docs then? I can't make sense to document a command that isn't present.